libraries/lib-project/CMakeLists.txt

```diff
@@ -19,10 +19,23 @@ set( SOURCES
    ProjectStatus.cpp
    ProjectStatus.h
 )
+
 set( LIBRARIES
    lib-registries-interface
    lib-xml-interface
 )
+
+
+if(${_OPT}enable_experimental_qt_support)
```

Comment:
This should go to a `lib-project-qml` or `lib-qml-support` or whatever. `lib-project` should remain green for all build configurations